vacuum-advisor/bloat.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: 'VACUUM Advisor: Bloat in Postgres'
+backlink_href: /docs/vacuum-advisor
+backlink_title: 'VACUUM Advisor'
+---
+
+In Postgres, bloat refers to a situation that the physical size of a table on
+disk is larger than its actual data size.
+
+## Why does bloat happen?
+
+In Postgres, any DELETEs or UPDATEs create dead rows in the tables they modify.
+VACUUM allows Postgres to reclaim these dead rows and reuse that space for
+future INSERTs or UPDATEs. Autovacuum is designed to perform VACUUM regularly in
+order to efficiently reuse space. This helps ensure that INSERTs and UPDATEs can
+use existing space instead of claiming new space. This helps prevent unnecessary
+growth of tables by utilizing existing space.
+
+### Insufficient VACUUMs
+
+If VACUUM is not performed with the appropriate frequency or at the right time,
+dead rows may not be reclaimed, causing the table to grow using new disk space
+even though there is available unused space.
+
+### Inefficient VACUUMs
+
+Even when VACUUM is performed sufficiently, it's possible that dead rows still
+cannot be reclaimed. This can happen when transactions are still accessing those
+dead rows, and the xmin horizon — represents the point up to which point VACUUM
+can clean up dead rows — is held back.
+As a result, VACUUM may not reclaim any or only a few dead rows, leading to bloat.
+
+
+## Why bloat is so bad
```

Comment:
Technically this works as a statement rather than a question, but I think having parallel construction in the titles (starting with "Why...") but having one as a question and another as a statement is kind of awkward. Also I think we should use a phrase a little more formal than "is so bad" :smile: . Maybe something like

```suggestion
## Why is bloat a problem?
```

Reply:
Thanks for this suggestion, I didn't like that title and but I wasn't able to come up something nice, so I was really seeking some suggestion 🙈